client/Lykke.Service.LoggingAdapter.Client/AutofacExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Autofac;
+using Common.Log;
+
+namespace Lykke.Service.LoggingAdapter.Client
+{
+    public static class AutofacExtension
+    {
+        public static void RegisterLoggingAdapterClient(this ContainerBuilder builder, string serviceUrl, ILog log)
+        {
+            if (builder == null) throw new ArgumentNullException(nameof(builder));
+            if (log == null) throw new ArgumentNullException(nameof(log));
+            if (string.IsNullOrWhiteSpace(serviceUrl))
+                throw new ArgumentException("Value cannot be null or whitespace.", nameof(serviceUrl));
+
+            builder.RegisterType<LoggingAdapterClient>()
```

Comment:
In case AutoRest client is used, to avoid memory leak need to use explicit HttpCLient parameter like this - https://github.com/LykkeCity/Lykke.Service.OperationsCache/blob/9bb38af321fddf90c45a793c5b48db941db40d65/client/Lykke.Service.OperationsCache.Client/OperationsCacheAPI.Extend.cs
But it's better to use Lykke.HttpClientGenerator